### DISCUSSION TITLE
EVALUACIONES

### DISCUSSION THREAD

**Discussion by carlos319** [rramosp/ai4eng.v1#289] [00 - General]
Buen dia @Felipe-RA, espero estes muy bien, tengo una pequeña duda sobre la evaluacion, dado de que si sera en clases debo solicitar un permiso en mi tranajo, ademas me reconfirmas la fecha si es tan amable. Gracias!

**Comment by Felipe-RA** [comment]
Hola @carlos319 

Disculpa la tardanza en responderte, se me había perdido esta notificación.

Las evaluaciones son en tiempo de clase. No de forma presencial.
